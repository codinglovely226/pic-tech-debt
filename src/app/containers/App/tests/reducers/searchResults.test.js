/* eslint-disable */

import searchResultsReducer from '../../reducers/searchResults';

import {
  logout,
  updateActionBarSearchResults,
  updateCaseListSearchResults,
  updateNewRecordSearchResults,
} from '../../actions';

describe('searchResultsReducer', () => {
  it('returns the initial state', () => {
    expect(searchResultsReducer(undefined, {})).toMatchSnapshot();
  });
  it('logout', () => {
    expect(searchResultsReducer({}, logout())).toMatchSnapshot();
  });
  it('update action bar search', () => {
    expect(searchResultsReducer({}, updateActionBarSearchResults(['Test data']))).toMatchSnapshot();
  });
  it('update case list search', () => {
    expect(searchResultsReducer({}, updateCaseListSearchResults(['Test data']))).toMatchSnapshot();
  });
  it('update new record search', () => {
    expect(searchResultsReducer({}, updateNewRecordSearchResults(['Test data']))).toMatchSnapshot();
  });
});
