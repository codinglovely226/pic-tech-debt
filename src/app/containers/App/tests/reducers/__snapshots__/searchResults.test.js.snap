exports[`searchResultsReducer logout 1`] = `
Object {
  "actionBar": Array [],
  "caseList": Array [],
  "newRecord": Array [],
}
`;

exports[`searchResultsReducer returns the initial state 1`] = `
Object {
  "actionBar": Array [],
  "caseList": Array [],
  "newRecord": Array [],
}
`;

exports[`searchResultsReducer update action bar search 1`] = `
Object {
  "actionBar": Array [
    "Test data",
  ],
  "caseList": Array [],
  "newRecord": Array [],
}
`;

exports[`searchResultsReducer update case list search 1`] = `
Object {
  "actionBar": Array [],
  "caseList": Array [
    "Test data",
  ],
  "newRecord": Array [],
}
`;

exports[`searchResultsReducer update new record search 1`] = `
Object {
  "actionBar": Array [],
  "caseList": Array [],
  "newRecord": Array [
    "Test data",
  ],
}
`;
